docs/documentation/configuration/index.mdx

```diff
@@ -1,115 +1,66 @@
 ---
-title: Indexing
+title: Create Index
 ---
 
-## Index Settings
-
-These settings can be set in `postgresql.conf` and affect system resources used during `CREATE INDEX`
-and normal `INSERT/UPDATE/COPY` statements.
-
-### Indexing Threads
-
-<Note>This setting requires superuser privileges.</Note>
-
-`paradedb.create_index_parallelism` sets the number of threads used during `CREATE INDEX`. The default is `8`. A value of zero will
-automatically detect "available parallelism" of the host computer.
+### Indexing Memory
 
-For best performance it is recommended to use one less than the total number of host computer cores.
+The default Postgres `maintenance_work_mem` value of `64MB` is quite conservative and can significantly slow down index builds. For large indexes,
+we recommend increasing `maintenance_work_mem`.
 
 ```sql
-SET paradedb.create_index_parallelism = 8;
+-- Should not exceed the server's available memory.
+SET maintenance_work_mem = '16GB';
 ```
 
-### Indexing Memory
+Generally speaking, this is the only setting that must be tuned to optimize index build times. The rest have sensible defaults.
```

Comment:
We discussed this in slack a bit.  We should recommend either an explicit value or some formula based on existing data size.

It's probably also worth mentioning here that we do index in parallel and the `maintenance_work_mem` is allocated across all the indexing threads.  IOW, it is not *per*-indexing-thread.

I see that is mentioned directly below on line 24, but might make sense to put it here too in case someone stops reading.